src/main/java/com/xiaomi/infra/pegasus/rpc/async/ClusterManager.java

```diff
@@ -130,6 +133,14 @@ public static Class getSocketChannelClass() {
     return metaList;
   }
 
+  public int getBackupRequestDelayMS() {
+    return backupRequestDelayMS;
+  }
+
+  public boolean isEnableBackupRequest() {
+    return backupRequestDelayMS > 0;
+  }
+
```

Comment:
So actually, "0" is also disable? This is different from previous definitions.

Reply:
What is the previous definitions?

Reply:
>  /**
     * The delay time of enable backup request. Defaults to {@literal -1}, which means there is no
     * need to send backup request, see {@link #DEFAULT_BACKUP_REQUEST_DELAY_MS}.
     *
You mean -1 is no need, and here means <= 0 is no need?

Reply:
I have already changed the code, and in my definition, <=0 is disable.